importer/markdown/src/test/java/org/itsallcode/openfasttrace/importer/markdown/TestMarkdownImporter.java

```diff
@@ -55,6 +51,20 @@ void testIdentifyTitle()
         assertMismatch(MdPattern.TITLE, "Title", "Title #", " # Title");
     }
 
+    @Test
+    void testIdentifyNeeds()
+    {
+        assertMatch(MdPattern.NEEDS_INT, "Needs: req, dsn", "Needs:req,dsn", "Needs:  \treq , dsn ");
+        assertMismatch(MdPattern.NEEDS_INT, "Needs:");
```

Comment:
We should wrap all of these with `assertAll(()-> ... )` so that all sub-asserts are always executed. Provides better debugging output.